### ISSUE TITLE
Android: VolumeManagerModule setupKeyListener NullPointerException error

### ISSUE THREAD

**Issue by arekm213** [hirbod/react-native-volume-manager#32]
We are getting a lot of error reports like this in our app which uses your library:

```
Exception java.lang.NullPointerException: Attempt to invoke virtual method 'android.view.Window android.app.Activity.getWindow()' on a null object reference
  at com.reactnativevolumemanager.VolumeManagerModule.lambda$setupKeyListener$1$com-reactnativevolumemanager-VolumeManagerModule (VolumeManagerModule.java:90)
  at com.reactnativevolumemanager.VolumeManagerModule$$ExternalSyntheticLambda2.run
  at android.os.Handler.handleCallback (Handler.java:958)
  at android.os.Handler.dispatchMessage (Handler.java:99)
  at android.os.Looper.loopOnce (Looper.java:205)
  at android.os.Looper.loop (Looper.java:294)
  at android.app.ActivityThread.main (ActivityThread.java:8248)
  at java.lang.reflect.Method.invoke
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:552)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:971)
```
  
Could you please think about proper fix for this?
We are currently early returning in `setupKeyListener` method if `mContext.getCurrentActivity()` returns `null`, but this probably could be handled better
This happens on `"react-native": "0.68.7"`, unfortunately im not able to reproduce this issue myself

**Comment by hgapsilva** [comment]
in the file `a/node_modules/react-native-volume-manager/android/src/main/java/com/reactnativevolumemanager/VolumeManagerModule.java b/node_modules/react-native-volume-manager/android/src/main/java/com/reactnativevolumemanager/VolumeManagerModule.java` , in the `setupKeyListener` method you can add a condition to see if the mContext exists, like this:

```
private void setupKeyListener() {
     runOnUiThread(() -> {

       if (mContext.getCurrentActivity() == null) return;

       View rootView =
       ... 
}
```

While you won't have this fix in the lib, you can make this change and then use this command in your project's terminal:

npx patch-package react-native-volume-manager

this will create a patch with the changes (the file shouldn't be large when created, it should only have the changes) and whenever you install node_modules it will use this patch to make your change.

this error is similar to this one:

https://github.com/hirbod/react-native-volume-manager/pull/28

**Comment by anirudhsama** [comment]
We are also seeing a lot of crashes due to this. Any chance it can be merged @hirbod ?

**Comment by hirbod** [comment]
I'll tackle this soon, I have lots of private things going on lately, sorry for the delay.

**Comment by anirudhsama** [comment]
Thank you!

**Comment by hirbod** [comment]
This is fixed in v2, I'll release it in ~30 minutes

**Comment by hirbod** [comment]
There was still a race condition, fixed in ^2.0.5
